quipucords/api/messages.py

```diff
@@ -55,6 +55,9 @@
 SOURCE_TYPE_INV = 'A source_type must not be provided when updating a source.'
 SOURCE_CRED_IDS_INV = 'Credential identifiers must be integer values.'
 SOURCE_MIN_CREDS = 'Source must have at least one set of credentials.'
+SOURCE_DELETE_NOT_VALID_W_SCANS = 'Source %s cannot be deleted '\
+    'because it is used by 1 or more scans.  '\
+    'First edit scans to remove source %s.'
```

Comment:
`First edit scans to remove source %s.` --> `First edit the following scans to remove the source: %s`